website/views.py

```diff
@@ -4823,7 +4826,8 @@ def chatbot_conversation(request):
         try:
             response = crc.invoke({"question": question})
         except Exception as e:
-            error_message = f"Error: {str(e)}"
+            logger.error(f"An error occurred: {str(e)}")
```

Comment:
```suggestion
            logger.error(f"An error occurred: {e}")
```
Or alternatively `logger.exception("An error occurred")`